search/include/pcl/search/search.h

```diff
@@ -418,7 +419,7 @@ namespace pcl
         std::string name_;
 
         /** \brief The number of threads to use when searching over multiple points or indices */
-        unsigned int num_threads_{1};
+        unsigned int num_threads_{0};
```

Comment:
I think this zero value is nowhere replaced by `omp_get_num_procs`? (Doesn't inherit from PCLBase)